src/AppBundle/Entity/Race.php

```diff
@@ -71,8 +71,10 @@ public function setRaceName($raceName)
 
     public function __toString()
     {
-        return $this->getRaceName();
+        if ($this->getRaceName() !== null) {
+            return $this->getRaceName();
+        } else {
+            return '';
```

Comment:
Race name is required ! 
No need that

Reply:
En fait, j'avais ajouté ça, pour que lorsque l'on clique sur add (affichage du formulaire de création) pour une race, il n'y ait pas d'erreur à cause de ToString